src/main/java/cart/controller/CartDeleteController.java

```diff
@@ -0,0 +1,24 @@
+package cart.controller;
+
+import cart.service.CartDeleteService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CartDeleteController {
+
+    private final CartDeleteService cartDeleteService;
+
+    public CartDeleteController(final CartDeleteService cartDeleteService) {
+        this.cartDeleteService = cartDeleteService;
+    }
+
+    @DeleteMapping("carts/{id}")
```

Comment:
```suggestion
    @DeleteMapping("/carts/{id}")
```

다른 엔드포인트와 동일하게 컨벤션을 맞춰주면 어떨까요?

Reply:
반영했습니다! 앞으로 더 신경쓰도록 하겠습니다 🥹